applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _UNICAST_CLIENT_H_
+#define _UNICAST_CLIENT_H_
+
+#include "le_audio.h"
+
+#include <zephyr/bluetooth/audio/audio.h>
+#include <audio_defines.h>
+
+enum unicast_discover_dir {
+	UNICAST_SERVER_SINK = BT_AUDIO_DIR_SINK,
+	UNICAST_SERVER_SOURCE = BT_AUDIO_DIR_SOURCE,
+	UNICAST_SERVER_BIDIR = (BT_AUDIO_DIR_SINK | BT_AUDIO_DIR_SOURCE)
+};
+
+#if CONFIG_BT_BAP_UNICAST_CONFIGURABLE
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                               \
+	BT_BAP_LC3_PRESET_CONFIGURABLE(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA,  \
+				       CONFIG_BT_AUDIO_BITRATE_UNICAST_SINK)
+
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE                                             \
+	BT_BAP_LC3_PRESET_CONFIGURABLE(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA,  \
+				       CONFIG_BT_AUDIO_BITRATE_UNICAST_SRC)
+
+#elif CONFIG_BT_BAP_UNICAST_16_2_1
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                               \
+	BT_BAP_LC3_UNICAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE                                             \
+	BT_BAP_LC3_UNICAST_PRESET_16_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#elif CONFIG_BT_BAP_UNICAST_24_2_1
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SINK                                               \
+	BT_BAP_LC3_UNICAST_PRESET_24_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+
+#define BT_BAP_LC3_UNICAST_PRESET_NRF5340_AUDIO_SOURCE                                             \
+	BT_BAP_LC3_UNICAST_PRESET_24_2_1(BT_AUDIO_LOCATION_FRONT_LEFT, BT_AUDIO_CONTEXT_TYPE_MEDIA)
+#else
+#error Unsupported LC3 codec preset for unicast
+#endif /* CONFIG_BT_BAP_UNICAST_CONFIGURABLE */
+
+/**
+ * @brief	Start service discover for the Bluetooth LE Audio unicast (CIS) server.
+ *
+ * @param[in]	conn	Pointer to the connection object.
+ * @param[in]	dir	Direction of the stream.
+ *
+ * @return 0 for success, error otherwise.
+ */
+int unicast_client_discover(struct bt_conn *conn, enum unicast_discover_dir dir);
+
+/**
+ * @brief	Handle a disconnection cleanup from the Bluetooth LE Audio unicast (CIS) client.
+ *
+ * @param[in]	conn	Pointer to the connection object.
+ */
+void unicast_client_conn_disconnected(struct bt_conn *conn);
+
+/**
+ * @brief	Start the Bluetooth LE Audio unicast (CIS) client.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int unicast_client_start(void);
+
+/**
+ * @brief	Stop the Bluetooth LE Audio unicast (CIS) client.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int unicast_client_stop(void);
+
+/**
+ * @brief	Send data from the LE Audio unicast (CIS) client, if configured as a sink.
```

Comment:
```suggestion
 * @brief	Send data from the LE Audio unicast (CIS) client, if at least one connected unicast server has a sink endpoint.
```
Probably too long, but something like this

Reply:
We probably shouldn't have ifs in the documentation, suggest just saying 
```suggestion
 * @brief	Send encoded audio.
```